src/NUnitFramework/framework/Internal/TestFilter.cs

```diff
@@ -94,7 +94,7 @@ public virtual bool IsExplicitMatch(ITest test)
         /// </summary>
         /// <param name="test">The test to which the filter is applied</param>
         /// <returns>True if the filter matches the an ancestor of the test</returns>
-        private bool MatchParent(ITest test)
+        public bool MatchParent(ITest test)
```

Comment:
I'll just trust you on this since I didn't see where it needed the access.
:-)

On Sun, Oct 4, 2015 at 8:10 AM, Neil Colvin <notifications@github.com>
wrote:

> In src/NUnitFramework/framework/Internal/TestFilter.cs
> <https://github.com/nunit/nunit/pull/862#discussion_r41098513>:
>
> > @@ -94,7 +94,7 @@ public virtual bool IsExplicitMatch(ITest test)
> >          /// </summary>
> >          /// <param name="test">The test to which the filter is applied</param>
> >          /// <returns>True if the filter matches the an ancestor of the test</returns>
> > -        private bool MatchParent(ITest test)
> > +        public bool MatchParent(ITest test)
>
> None, that I am aware of. But that does not really make a difference. It
> could, in the future.
>
> And we do need to access it here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/862/files#r41098513>.
>